### ISSUE TITLE
crio array push is not a function

### ISSUE THREAD

**Issue by ArielGueta** [planttheidea/crio#49]
Hey, I am trying to push new value to crio array, and it throws.

```
const foo = crio(['foo']);
const fooBar = foo.push('bar');
console.log('foo', fooBar);
```

> TypeError: foo.push is not a function

**Comment by planttheidea** [comment]
Well that is an embarassing regression ... the one method I missed in the v4 rewrite happens to be the exact method I use in my example. My apologies, egg is all over my face.

I shall have a fix for this shortly.

**Comment by planttheidea** [comment]
Fixed in `4.1.2`
